faq.md

```diff
@@ -1,15 +1,24 @@
-### What is the difference between 18F Google Analytics and the Digital Analytics Program?  
-...
+### What is the difference between 18F Google Analytics and the Digital Analytics Program (DAP)?
 
-### Can I give access to a project's analytics to a client?  
+The [Digital Analytics Program](http://www.digitalgov.gov/services/dap/) and 18F Google Analytics are different services with two distinct code snippets.
```

Comment:
@ramirezg  -- this is better, but still needs more clarification. Would folks participate in both or just one? Could folks participate in both?